dev-util/gn/gn-9999.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit ninja-utils python-any-r1 toolchain-funcs
+
+DESCRIPTION="GN is a meta-build system that generates build files for Ninja"
+HOMEPAGE="https://gn.googlesource.com/"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gn.googlesource.com/gn"
+else
+	# The version number is derived from `git describe HEAD --abbrev=12`
```

Comment:
Link to the wiki page you've started as well, maybe?

Reply:
Let's come back to that in a little bit - I may as well start bumping gn more frequently and I want to get the docs up-to-date and under the Chromium Project namespace so that Chromium/Chrome/Tooling version bumps are actually documented in a way that makes sense.